ruby-challenges.rb

```diff
@@ -31,3 +48,14 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def checking_palindrome(string)
+  if string == string.reverse
+    "#{string} is a palindrome"
+  else "#{string} is not a palindrome"
+  end
+end
+  p checking_palindrome(is_palindrome1)
+  p checking_palindrome(is_palindrome2.downcase)
+  p checking_palindrome(is_palindrome3)
```

Comment:
I would suggest placing a downcase inside your method.  Currently, Racecar and Rotator would return that they are _not_ a palindrome since they have a capital letter in them.